### DISCUSSION TITLE
The Architecture

### DISCUSSION THREAD

**Discussion by williamluisan** [aasumitro/posbe#21] [Q&A]
![Screenshot_120822_082558_PM](https://user-images.githubusercontent.com/22985237/206457886-f1a25f32-a2c4-4f5d-9ebf-5b9d4bf05f83.jpg)

Mit, is this the bigger picture of this software architecture you're trying to achieve?
@aasumitro

**Comment by aasumitro** [comment]
too complex to be true 🤣

**Reply by aasumitro** [reply]
*in my mind

**Reply by williamluisan** [reply]
Ya, but I saw that you're kind like developing and use an API Gateway in this project, is that correct?

**Reply by aasumitro** [reply]
api gateway? nope, probably in the future

**Reply by williamluisan** [reply]
Atiq kira mirip2 API gateway, hahahaha

**Reply by aasumitro** [reply]
nda guna soalnya klo for skarang
